src/m365/pp/commands/managementapp/managementapp-list.ts

```diff
@@ -0,0 +1,36 @@
+import { Logger } from '../../../../cli';
+import request from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+class PpManagementAppListCommand extends GraphCommand {
```

Comment:
Since we're not using Graph, let's introduce a separate base class for Power Platform command with the `https://api.bap.microsoft.com` set as resource. In the future, we can then refactor it to return the correct URL based on the cloud/environment.

Reply:
And addition; since we have a list command I assume we should create a `PowerPlatformItemsListCommand` and for the Add command we should have a `PowerPlatformCommand`. Since the `GraphListCommand` extends the `GraphCommand` we should do the same for `PowerPlatformCommand` implementation. However that means we either have to duplicate our `resource` implementation or our `getAllItems` implementation.